tests/lldp/test_lldp.py

```diff
@@ -33,11 +33,15 @@ def test_lldp(duthosts, enum_rand_one_per_hwsku_frontend_hostname, localhost,
         # Compare the LLDP neighbor name with minigraph neigbhor name (exclude the management port)
         assert v['chassis']['name'] == config_facts['DEVICE_NEIGHBOR'][k]['name']
         # Compare the LLDP neighbor interface with minigraph neigbhor interface (exclude the management port)
-        assert v['port']['ifname'] == config_facts['DEVICE_NEIGHBOR'][k]['port']
+        if request.config.getoption("--neighbor_type") == 'eos':
```

Comment:
Will the default value for --neighbor_type be 'eos' if we do not pass this option in pytest?

Reply:
Yes, the default is eos